config/defaults.js

```diff
@@ -122,5 +132,8 @@ module.exports = {
     password: process.env.TESTDROID_PASSWORD
   },
 
-  dockerWorkerPrivateKey: '/etc/docker-worker-priv.pem'
+  dockerWorkerPrivateKey: '/etc/docker-worker-priv.pem',
+
+  sslCertificate: '/etc/star_taskcluster-worker_net.crt',
+  sslKey: '/etc/star_taskcluster-worker_net.key'
```

Comment:
Nit, but maybe move this under `logging`.

Or under `ssl.crtFile` and `ssl.keyFile`... I guess we'll use it for more than logging long term anyways.